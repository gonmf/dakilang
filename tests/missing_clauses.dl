

alpha(A, B) :- missing(A, B).
alpha(A, B)?

beta(X) :- missing(X, X, X).
beta(123)?

charlie(123)?

delta(1, 2).

delta(1)?
delta(X)?

