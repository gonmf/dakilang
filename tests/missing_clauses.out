>
>
> alpha(A, B) :- missing(A, B).
> alpha(A, B)?
No

>
> beta(X) :- missing(X, X, X).
> beta(123)?
No

>
> charlie(123)?
No

>
> delta(1, 2).
>
> delta(1)?
No

> delta(X)?
No

>
